wordpress-mobile/react-native-reanimated
# Fix crash related to Android layout reanimation due to `NullPointerException`

In order to address https://github.com/wordpress-mobile/gutenberg-mobile/issues/4801, this PR incorporates the fix from [this upstream commit](https://github.com/software-mansion/react-native-reanimated/commit/e039995361699a5a94162f1dac058c639f87261f) into the `wp-fork` branch.

**How to test:**
1. Check out the branch `rnmobile/fix/drag-and-drop-android-crash-when-closing-editor` in the Gutenberg repository.
2. Open the app and connect it to the Metro server.
3. Start dragging a block.
4. While moving the block around the screen, tap the back button with another finger to close the editor.
5. Observe that the app doesn't crash.
